### ISSUE TITLE
Are these types actually equivalent

### ISSUE THREAD

**Issue by Lysxia** [Gabriella439/Haskell-Lens-Tutorial-Library#8]
I've seen some people confused by this excerpt about the type of `Getting`:

    type Getting b a b  = (b -> Const b b) -> (a -> Const b a)

     -- ... equivalent to: (b ->       b  ) -> (a ->       b  )

     -- ... equivalent to:                     (a ->       b  )

But the last one isn't an equivalence.

One is `(forall r. Getting r a b) <-> (a -> b)`.

**Comment by Gabriella439** [comment]
Yeah, that is not equivalent.  I'll rephrase it to avoid confusion
